app/routes.js

```diff
@@ -1,6 +1,31 @@
-var express = require('express'),
-    router  = express.Router(),
-    _       = require('underscore');
+var express 	= require('express'),
+    router  	= express.Router(),
+    fs          = require('fs'),
+	basicAuth 	= require('basic-auth'),
```

Comment:
would be best to use the same convention of using spaces to prevent indentation mess